packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx

```diff
@@ -1,13 +1,14 @@
-import { ChangeEvent, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import Blockies from "react-blockies";
 import { useEnsAddress, useEnsAvatar, useEnsName } from "wagmi";
 import { isAddress } from "ethers/lib/utils";
+import { InputBase } from "./InputBase";
 
-type TAddressInputProps = {
-  onChange?: (arg: string) => void;
-  placeholder?: string;
+type AddressInputProps = {
```

Comment:
I was wondering if we should make a common interface as : 
```js 
Interface IInput {
  value: string;
  onChange: (newValue: string) => void;
  name?: string;
  placeholder?: string;
}
```
and extend other Inputs like `AddressInput`, `Bytes32Input` etc from it because most of them have same properties : `name`, `placeholder` etc

Reply:
Good point! Made a common input props interface.

(As a sidenote, I think we shouldn't prefix interfaces or types with I or T. Single letter prefixes are, in my experience, usually reserved for generic types)